### DISCUSSION TITLE
How to stop poles from being detected as people?

### DISCUSSION THREAD

**Discussion by joseror** [blakeblackshear/frigate#6187] [Ask A Question]
Hi, I have two poles in my garage that keep being detected as a person. I added some people masks, but they keep coming up. I had to lower my threshold to 40% due to false negatives and that increased the issue a bit. Any ideas on how to avoid this from happening?

![person2](https://user-images.githubusercontent.com/49996350/233497413-45143907-bc0a-497b-8890-bd81e3fd0568.png)
![person1](https://user-images.githubusercontent.com/49996350/233497415-d0ab3627-9e02-4bff-93ed-cce64f7ef006.png)
![mask1](https://user-images.githubusercontent.com/49996350/233497416-28c982b2-9161-460d-855b-a48783e24966.png)
![mask2](https://user-images.githubusercontent.com/49996350/233497418-b0c68114-594c-4a8a-b311-63645c3c760b.png)

**Comment by NickM-27** [comment]
this is confusing. why would you lower your threshold for false positives, that would only increase the number of false positives. 

also you should be aware that masks are evaluated based on the bottom center pixel of the bounding box. all you need is a small box at the bottom of the pole, which is the one part that seems to be missing

**Reply by joseror** [reply]
I lowered it because some times people events were not being recorded when they were on one of the corners of the stream. Lowering the threshold solved that issue. Either way, the poles usually come as people with a score over 70% so they would be picked up either way.

**Reply by NickM-27** [reply]
right, that would be a false negative though. Just wanted to make sure

**Reply by joseror** [reply]
> this is confusing. why would you lower your threshold for false positives, that would only increase the number of false positives.
> 
> also you should be aware that masks are evaluated based on the bottom center pixel of the bounding box. all you need is a small box at the bottom of the pole, which is the one part that seems to be missing

So I need to modify the masks that I attached to just cover the bottom of the pole? Something like this?
![example](https://user-images.githubusercontent.com/49996350/233501978-97d317be-57cc-42f7-b844-47205ccc1875.png)

**Reply by NickM-27** [reply]
yes, although I would make the bottom edge be further down just a bit

**Reply by joseror** [reply]
> yes, although I would make the bottom edge be further down just a bit

Would it make a different if I mask the whole pole including an extra bit at the bottom? Earlier today just the top half the pole was detected as a person so I guess just covering the base wouldn't cover that instance.

**Reply by NickM-27** [reply]
yeah that would be fine most likely

**Comment by FrigateEnthusiast** [comment]
Since "person riding a unicycle" is not one of the canned objects, have you considered leaning a spare tire against the pole? :)
